### PR TITLE
Editorial: Export several terms needed for the File System Access API.

### PR DESCRIPTION
**Pull request by mkruisselbrink**
Also renames "entry" to "file system entry", as the former would be too
generic of a name to export. Similarly terms that used to have for=entry
will now have for=fsentry. IDs were preserved where it seemed sensible.

***
<a href="https://whatpr.org/fs/8.html" title="Last updated on Nov 15, 2022, 1:01 AM UTC (3742355)">Preview</a> | <a href="https://whatpr.org/fs/8/f4fab47...3742355.html" title="Last updated on Nov 15, 2022, 1:01 AM UTC (3742355)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -85,36 +83,37 @@ Issue: We should consider having further normative restrictions on file names th
 never be allowed using this API, rather than leaving it entirely up to underlying file
 systems.
 
-A <dfn lt="file|file entry">file entry</dfn> additionally consists of
-<dfn for="file entry">binary data</dfn> (a [=byte sequence=]) and a
+A <dfn export id=file>file entry</dfn> additionally consists of
+<dfn for="file entry" export>binary data</dfn> (a [=byte sequence=]) and a
 <dfn for="file entry">modification timestamp</dfn> (a number representing the number of milliseconds since the <a spec=FileAPI>Unix Epoch</a>).
```
Comment (annevk) [inline index.bs:88]: Should this be exported?

File: index.bs [hunk 2]
```diff
@@ -85,36 +83,37 @@ Issue: We should consider having further normative restrictions on file names th
...
 
-A <dfn lt="directory|directory entry">directory entry</dfn> additionally consists of a [=/set=] of
-<dfn for="directory entry">children</dfn>, which are themselves [=/entries=]. Each member is either a [=/file=] or a [=directory=].
+A <dfn export id=directory>directory entry</dfn> additionally consists of a [=/set=] of
+<dfn for="directory entry">children</dfn>, which are themselves [=/file system entries=].
```
Comment (annevk) [inline index.bs:91]: And this?

**Review by annevk — COMMENTED** [review]
Apologies for the delay. I pushed a fixup that makes this more consistent with other WHATWG documents and also preserves more IDs.

File: index.bs (hunk 2 above)
Comment (a-sully) [reply index.bs:131]: Nope, I don't think so. It's not used in the PR I just put up to remove duplication, at least https://github.com/WICG/file-system-access/pull/392/

File: index.bs (hunk 1 above)
Comment (a-sully) [reply index.bs]: Yes, it's now exported

**Review by a-sully — APPROVED** [review]

**Comment by a-sully** [comment]
Merging this so we can land https://github.com/WICG/file-system-access/pull/392 and finally check off the last box on https://github.com/WICG/file-system-access/issues/342 🎉

**Pull request merged**